home-task-1/src/App.js

```diff
@@ -1,25 +1,60 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className='App'>
-      <header className='App-header'>
-        <img src={logo} className='App-logo' alt='logo' />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className='App-link'
-          href='https://reactjs.org'
-          target='_blank'
-          rel='noopener noreferrer'
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React, { Component, PureComponent, createElement } from 'react';
+
+class HeaderTitle extends PureComponent {
+  render() {
+    return <h1>{this.props.title}</h1>;
+  }
+}
+
+const FinalCounter = ({ counter }) => <h2>counter: {counter}</h2>;
+
+const ButtonCounter = ({ handle, text }) =>
+  createElement('button', { onClick: handle }, text);
+
+class App extends Component {
+  constructor() {
+    super();
```

Comment:
You don't need to call `super`
Example: [React.Component](https://reactjs.org/docs/react-component.html#overview)